src/pages/solve/FeatureModelAnomaly.tsx

```diff
@@ -22,31 +33,49 @@ const FeatureModelAnomaly: NextPage = () => {
     useState<Option[]>(anomalies);
 
   return (
-    <TextInputMask
-      title="Feature Model Anomaly Solver"
-      description="For a tree of features with cross tree constraints, the solver detects anomalies."
-      textPlaceholder="Enter your feature model in UVL format"
-      onTextChanged={setUvl}
-      body={
-        <VStack>
-          <Box width="300px">
-            <MultiSelect
-              options={anomalies}
-              value={selectedAnomalies}
-              onChange={setSelectedAnomalies}
-              labelledBy="Select anomalies"
-            />
-          </Box>
+    <Layout>
+      <Heading as="h1">FeatureModel Anomaly Solvers</Heading>
+      <Text color="text" align="justify">
+        For a tree of features with cross tree constraints, these solvers can
+        detect various anomalies. Feature Models are given in the Universal
+        Variablity language.
```

Comment:
Imo we should add the abbreviation, UVL, here. People might only be familiar with that.

Reply:
We also support Extended UVL btw